pkg/resource/write/writer.go

```diff
@@ -65,7 +65,7 @@ func (this *resourceWriter) AddResources(resources []resource.KubernetesResource
 		requested := resources[index]
 		if this.ownerRefs != nil {
 			requested.SetOwnerReferences(this.ownerRefs)
-		} else if this.ownerController != nil {
+		} else if this.canSetOwnerRef(requested, this.ownerController) {
```

Comment:
Is that because cluster-scoped objects can't have owners? Or they can't have owners that are namespace-scoped/